website/static/js/profile.js

```diff
@@ -581,6 +602,39 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+
+    self.institutionsEmpty = ko.computed(function() {
```

Comment:
Institution is the name field for both jobs and schools.  So I would think this would be acceptable if we stick to using the list view for only jobs and schools.  At some point all of the profile.js should probably be rewritten, and I would suggest we look at the jobs and schools objects at that time.  LMK if you think otherwise.  :fist: